src/notation/internal/notationnoteinput.h

```diff
@@ -92,6 +92,8 @@ class NotationNoteInput : public INotationNoteInput, public async::Asyncable
     async::Notification m_stateChanged;
     async::Notification m_noteAdded;
 
+    EngravingItem* m_lastInputedItem = nullptr;
```

Comment:
Ms::InputState already stores the position of the last entered note, so this pointer looks redundant. Maybe we don't use it correctly?

Reply:
Oh okay. I didn't know about that